backend/experiment/models.py

```diff
@@ -145,7 +146,7 @@ def consent_upload_path(instance: Experiment, filename: str) -> str:
     folder_name = experiment.slug
     language = instance.language
 
-    return f"consent/{folder_name}/{language}-{filename}"
+    return join('consent', folder_name, language, filename)
```

Comment:
```suggestion
    return join('consent', folder_name, f"{language}-{filename}")
```
Note that your replacement is different, since the current setup writes a `en-consent.md` file, not to a folder `en` with `consent.md`.